### ISSUE TITLE
Error getting sensitive fields

### ISSUE THREAD

**Issue by YaraslauZhylko** [MethodFi/method-python#23]
In the recently released version `0.0.34`, when trying to get sensitive values in the dev environment, SDK is raising an error:
> Field "fields" is required. Verify your request and try again.

```python
from backend.settings import Settings
from method.errors import MethodInvalidRequestError

method_sdk = Method(env="dev", api_key="*****")
try:
    method_sdk.entities.get_sensitive_fields("ent_A6nGi7Vfdzzxe")
except MethodInvalidRequestError as err:
    err.code  # 400
    err.message  # 'Field "fields" is required. Verify your request and try again.'

Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File ".venv/lib/python3.12/site-packages/method/resources/Entity.py", line 331, in get_sensitive_fields
    return super(EntityResource, self)._get_with_sub_path('{_id}/sensitive'.format(_id=_id))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.12/site-packages/method/errors.py", line 37, in wrapper
    raise MethodError.generate(res['error'])
method.errors.MethodInvalidRequestError
```

**Comment by YaraslauZhylko** [comment]
Seems like we need to provide the fields explicitly.
This can works in dev:
```python
 >>> method_sdk.entities._get_with_sub_path('{_id}/sensitive?fields[]=dob&fields[]=address'.format(_id="ent_kHAqhRjQVVVpr"))
 {
     'first_name': None,
     'last_name': None,
     'phone': None,
     'phone_history': [],
     'email': None,
     'dob': '2000-01-01',  # <-----
     'address': {  # <-----
         'line1': '705 Blanton Dr',
         'line2': None,
         'city': 'Sherman',
         'state': 'TX',
         'zip': '75092'
     },
     'address_history': [],
     'all_addresses': [],
     'ssn_4': None,
     'ssn_6': None,
     'ssn_9': None,
     'identities': []
 }

**Comment by ossiggy** [comment]
@YaraslauZhylko thanks for flagging this.  The only issue I see with providing the fields explicitly is that there can be more or less fields depending on specific use cases, so hard coding them all does not seem feasible.  I will work with the team to figure this out!

**Comment by YaraslauZhylko** [comment]
Here's a a potential fix: https://github.com/MethodFi/method-python/pull/24

@ossiggy  It just allows to pass any value and API will return the error if a field is now allowed. See the test examples in the PR.

**Comment by YaraslauZhylko** [comment]
Seems to be fixed in `0.0.35`.
We can close this issue.
